src/main/java/io/jmix/uisamples/entity/Document.java

```diff
@@ -0,0 +1,36 @@
+package io.jmix.uisamples.entity;
+
+import io.jmix.core.FileRef;
+import io.jmix.core.entity.annotation.JmixGeneratedValue;
+import io.jmix.core.entity.annotation.JmixId;
+import io.jmix.core.metamodel.annotation.JmixEntity;
+import io.jmix.core.metamodel.annotation.JmixProperty;
+
+import java.util.UUID;
+
+@JmixEntity
```

Comment:
Please look other entities declaration. @Entity and @Table annotations are missing here